docs/api/dialog.md

```diff
@@ -32,9 +32,17 @@ The `dialog` module has the following methods:
   * `buttonLabel` String (optional) - Custom label for the confirmation button, when
     left empty the default label will be used.
   * `filters` [FileFilter[]](structures/file-filter.md) (optional)
-  * `properties` String[] (optional) - Contains which features the dialog should use, can
-    contain `openFile`, `openDirectory`, `multiSelections`, `createDirectory`
-    and `showHiddenFiles`.
+  * `properties` String[] (optional) - Contains which features the dialog should
```

Comment:
Multiple values can be specified here, right? How is this property string delimited? Spaces?